pkg/rtc/room.go

```diff
@@ -777,10 +765,8 @@ func (r *Room) onTrackPublished(participant types.LocalParticipant, track types.
 		onParticipantChanged(participant)
 	}
 
-	notifier := r.changeNotifier.GetNotifier(string(track.ID()))
-	if notifier != nil {
-		notifier.NotifyChanged()
-	}
+	r.trackManager.AddTrack(track, participant.Identity(), participant.ID())
+	r.trackManager.NotifyTrackChanged(track.ID())
```

Comment:
This is happening both here and inside `AddTrack` itself. Is that intentional?

Reply:
good catch.. will remove the duplicate